File: 프로그래머스/level1_소수 찾기.py
def is_prime(n):
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True

def solution(n):
    answer = 0
    for i in range(2, n + 1):
        if is_prime(i):
            answer += 1

    return answer

print(solution(5))

File: 프로그래머스 복습(java)/src/level1/level1_소수_찾기.java
package level1;

public class level1_소수_찾기 {
    static class Solution {
        public Boolean isPrime(int n) {
            for (int i = 2; i < Math.sqrt(n) + 1; i++) {
                if (n % i == 0) {
                    return false;
                }
            }
            return true;
        }

        public int solution(int n) {
            int answer = 0;
            for (int i = 1; i < n + 1; i++) {
                if (isPrime(i)) {
                    answer++;
                }
            }
            return answer;
        }
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        System.out.println(solution.solution(10));
        System.out.println(solution.solution(5));
    }
}
